PythonTool/TagExpansion.py

```diff
@@ -50,7 +45,9 @@ def get_vector_tags(mb, element):
     Returns:
     ________
        element_list: List
```

Comment:
for all of these, specify the data type in the list. Is this one a list of entity handles?